src/main/webapp/scripts/cookies.js

```diff
@@ -0,0 +1,85 @@
+const HOURS_PER_DAY = 24;
```

Comment:
These kinda seem like Juan's changes on top of your own in one pull request. You might want to re-create the pull request and change the base branch.

Reply:
My bad, thought I did that.